CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## [1.137.2] - 2024-04-05
+
+### Features / Changes
+
+- Show reactions in summaries ([#5387](https://github.com/deltachat/deltachat-core-rust/pull/5387)).
+
+### Fixes
+
+- Do not show empty summary if message reacted to is deleted.
```

Comment:
do we need to mention fixes for bugs that were introduced by changes that were made after the last version?